src/LinAlg/SAM.C

```diff
@@ -939,3 +939,23 @@ bool SAM::getNodalReactions (int inod, const Vector& rf, Vector& nrf) const
 
   return haveRF;
 }
+
+
+std::set<int> SAM::getEquations(char dofType, int dof) const
+{
```

Comment:
Det er typedef'd en IntSet i SAM.h som brukes for andre std::set<int> objekter i SAM scope.

Ville ellers implementert denne metoden mer direkte, uten å gå via andre metoder for å aksessere data members som det er full tilgang til, f.eks. slik:

IntSet result;
for (int inod = 0; inod < nnod; inod++)
  if (dofType = nodeType[inod] || (dofType == 'D' && nodeType[inod] == ' ')) {
    if (dof > 0) {
      int idof = madof[inod] + dof-1;
      int ieq = idof < madof[inod+1] ? meqn[idof-1] : 0;
      if (ieq > 0) result.insert(ieq);
    }
    else for (int idof = madof[inod]; idof < madof[inod+1]; idof++) {
      int ieq = meqn[idof-1];
      if (ieq > 0) result.insert(ieq);
    }
  }